vllm/tests/docker/docker-compose.yaml

```diff
@@ -0,0 +1,11 @@
+version: "3.9"
+services:
+  caddy:
+    image: caddy:2.7
+    container_name: caddy
+    ports:
+      - "8000:8000"
+    volumes:
+      - ./Caddyfile:/etc/caddy/Caddyfile
+      - ../fixtures/vllm_metrics.txt:/usr/share/caddy/metrics
+      - ../fixtures/vllm_version.json:/usr/share/caddy/version
```

Comment:
I _think_ these can be combined, i.e. we just mount the into the `/usr/share/caddy` dir.

Reply:
Yea they probably can. It was my first time working with Caddy files so I needed to be explicit with where everything was mounted to solidify my understanding of how they worked is all. 👍

Reply:
I tried it out and it didn't work. not gonna invest more into this.